src/opa.js

```diff
@@ -117,7 +161,7 @@ async function _load_policy(policy_wasm, memory) {
     });
 
     const builtins = _dumpJSON(wasm.instance, memory, wasm.instance.exports.builtins());
-
+    console.log({builtins})
```

Comment:
I think you already mentioned it, but we'd probably want to remove the log message.

Reply:
Fixed